tasks/e2e/cypress/e2e/01-tutorial/tutorial.cy.js

```diff
@@ -17,10 +17,10 @@ import {
   // test_auth_cell_failure,
 } from '../01-tutorial/sharedTests'
 
-import Step0_1_RedwoodToml from './codemods/Step0_1_RedwoodToml'
-import Step0_2_GraphQL from './codemods/Step0_2_GraphQL'
-import Step1_1_Routes from './codemods/Step1_1_Routes'
-import Step9_3_DisableAuth from './codemods/Step9_3_DisableAuth'
+// import Step0_1_RedwoodToml from './codemods/Step0_1_RedwoodToml'
+// import Step0_2_GraphQL from './codemods/Step0_2_GraphQL'
+// import Step1_1_Routes from './codemods/Step1_1_Routes'
+// import Step9_3_DisableAuth from './codemods/Step9_3_DisableAuth'
```

Comment:
These are unused right? Easier to tell in editor but if I remember correctly they're just grayed out

Reply:
Yeah I could have removed them but the where they had been used was commented out so I just commented out the unused imports. Purely a "remove the red squiggles" move